Intersect.Client/Entities/Entity.cs

```diff
@@ -1386,6 +1386,147 @@ public virtual void DrawName(Color textColor, Color borderColor = null, Color ba
             );
         }
 
+        // Tags
+        public virtual void DrawTag(NpcTagPos npcTagPos, PlayerTagPos playerTagPos)
+        {
+            // Variables
+            string tagFileName = "";
+            string entityName = this.Name;
+            bool customNpcTag = Options.Npc.CustomTagIcons.Contains(entityName);
+            bool customPlayerTag = Options.Player.CustomTagIcons.Contains(entityName);
+            bool customNpcTagOnly = Options.Npc.ShowCustomTagsOnly;
+            var nameSize = Graphics.Renderer.MeasureText(entityName, Graphics.EntityNameFont, 1);
+            var nameCentHorPos = (int)Math.Ceiling(GetCenterPos().X);
+            var nameVertPos = GetLabelLocation(LabelType.Name);
+            float x;
```

Comment:
Combine the `x` and `y` declarations into a single line.

Reply:
👍🏽 - Not sure if this is what you mean, this is how it is now:

`float x, y;`